src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -119,6 +121,22 @@ public String getPreviousReleaseVersion() {
     public class GitHub {
 
         /**
+         * GitHub API endpoint address, for example:  https://api.github.com/
+         */
+        public String getApiUrl() {
+            return getString("gitHub.apiUrl");
+        }
+
+        /**
+         * See {@link #getApiUrl()}
+         *
+         * @param apiUrl GitHub (or gitHub enterprise) API endpoint address, for example: https://api.github.com/
+         */
```

Comment:
Maybe a class for all GitHub URLs. It may be useful, since some parameters are repeated, like access_token or per_page